cubic-server/world-storage/WorldStorage.hpp

```diff
@@ -0,0 +1,28 @@
+#include <unordered_map>
+#include "LevelData.hpp"
+#include "Level.hpp"
+
+#ifndef WORLD_WORLD_HPP
+#define WORLD_WORLD_HPP
+
+class WorldStorage
```

Comment:
Ok, this may be stupid but why does this need to be separated from the World and Dimension class ?
From what I see here, this is just a data class for storing the levelData, and the levels that are for the dimension

So why not storing this in the existing class ?

Reply:
I think it would make sense, as there are many helper functions you can have in this.
And you can also later on add functions just to load WorldStorage from the disk and bind it to a World.
Which could allow blockless worlds (For example to implement a queue, see [NanoLimbo](https://github.com/Nan1t/NanoLimbo)).